src/d_deh.c

```diff
@@ -3383,11 +3459,13 @@ char *dehReformatStr(char *string)
 //
 // killough 10/98: only strip at end of line, not entire string
 
-void lfstrip(char *s)  // strip the \r and/or \n off of a line
+static void lfstrip(char *s) // strip the \r and/or \n off of a line
 {
-  char *p = s+strlen(s);
-  while (p > s && (*--p=='\r' || *p=='\n'))
-    *p = 0;
+    char *p = s + strlen(s);
+    while (p > s && (*--p == '\r' || *p == '\n'))
+    {
```

Comment:
`*p = '\0'`

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/2146/commits/263498d67a9c44fd80df539f051dd5eff720d3e3